### PR TITLE
AWS

### PR DESCRIPTION
**Pull request by khorne3**
First set of AWS updates

### CODE REVIEW

File: articles/integrations/aws-api-gateway-2/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+description: blah blah
```
Comment (mpaktiti) [inline articles/integrations/aws-api-gateway-2/index.md]: Maybe something a bit more descriptive? 😄

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Hey Katie, the heroku app does not work. I can see this in the logs:

```
Error compiling document articles/integrations/aws-api-gateway-2/index.md: 
Couldn't resolve include ./_stepnav included from file /app/docs/articles/integrations/aws-api-gateway-2/index.md
```

**Comment by khorne3** [comment]
@mpaktiti Whoops, sorry about that. I wanted to move all things API Gateway rewrite out and I missed 1-2 commits when cherry picking.

File: articles/integrations/aws/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (mpaktiti) [inline articles/integrations/aws/index.md:1]: Good idea adding an index! Can you convert this to the `index.html` format though? It looks a bit better that way :)

Comment (mpaktiti) [reply articles/integrations/aws/index.md:1]: And I would simplify the title to `Integrate Auth0 with Amazon Web Services (AWS)`

File: articles/integrations/aws/index.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+description: How to integrate AWS with Auth0 to setup SSO or obtain AWS tokens.
+---
+
+# Auth0-Amazon Web Services (AWS) Integrations
+
+Auth0 supports integration with AWS' Identity and Access Management (IAM) service, allowing you to:
+
+ * [Perform SSO with the AWS Dashboard](/aws/sso)
```
Comment (mpaktiti) [inline articles/integrations/aws/index.md]: The first two links are broken, you should either update them including the `/integrations` part of add url metadata at the docs.

File: articles/integrations/aws/sso.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
+# Enable SSO to the AWS Console
```
Comment (mpaktiti) [inline articles/integrations/aws/sso.md:6]: Missing `description` and `toc` metadata

File: articles/integrations/aws/aws-api-setup.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+![](/media/articles/tutorials/aws/provider-summary.png)
+
+### Role ARN
+
+In the IAM console, navigate to [Roles](https://console.aws.amazon.com/iam/home#/roles). Select the role in which you're interested to open up its summary page. Copy the **Role ARN** value, which is listed first under **Summary**.
+
+![](/media/articles/tutorials/aws/role-summary2.png)
+
+### Keep Reading
```
Comment (mpaktiti) [inline articles/integrations/aws/aws-api-setup.md]: use `:: next steps`

File: articles/integrations/aws/aws-api-setup.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
+---
+description: How to enable delegated authentication with AWS APIs.
+url: /aws-api-setup
+toc: true
+---
```
Comment (mpaktiti) [inline articles/integrations/aws/aws-api-setup.md]: Maybe it's me but I didn't understand the business case or how to test this configuration

File: articles/integrations/aws/tokens.md [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+
+::: warning
+Newly-created clients may not use [legacy grant types](/clients/client-grant-types), which are required for use with the [Delegation endpoint](/api/authentication#get-token-info). If you have any questions about which alternative you should use, please contact [Support](${env.DOMAIN_URL_SUPPORT}).
+:::
+
+Auth0 integrates with the AWS Security Token Service (STS) to obtain an limited-privilege credentials for AWS Identity and Access Management (IAM) users or for users that you authenticate (federated users). These credentials can then be used to call the AWS API of any Auth0-supported [identity provider](/identityproviders).
+
+## Sample Configuration
+
+![](/media/articles/integrations/aws/aws-sts.png)
```
Comment (mpaktiti) [inline articles/integrations/aws/tokens.md:16]: Send the diagram to design

File: articles/integrations/aws/tokens.md [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+
+1. The web app authenticates its users via Social providers, such as Facebook, LinkedIn, or Twitter, or corporate credentials, such as Active Directory, Office365, or Salesforce.
```
Comment (mpaktiti) [inline articles/integrations/aws/tokens.md]: maybe add some links to our individual identity provider docs (facebook, etc)

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Hey Katie, some changes required. Thanks!

**Comment by mpaktiti** [comment]
@khorne3 While you are at it add also an update feed entry

**Pull request merged**